cmd/aro/operator.go

```diff
@@ -81,6 +82,11 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 		return err
 	}
 
+	env, err := env.NewEnv(ctx, log)
```

Comment:
I think this is wrong place to be here. This instantiates `envCore`  which has a bunch of stuff which should never land into operator context. Basically, anything in env Core should not be present in the customer cluster.

You need this only to get ACR image for the workaround. Need to think more about how to add outside env package.